src/dp_cntrack.c

```diff
@@ -162,6 +162,7 @@ static __rte_always_inline struct flow_value *flow_table_insert_entry(struct flo
 	flow_val->timeout_value = flow_timeout;
 	flow_val->created_port_id = port->port_id;
 
+	// what if key->l3_type == RTE_ETHER_TYPE_IPV6?
 	pfx_ip.ip_type = RTE_ETHER_TYPE_IPV4;
 	pfx_ip.ipv4 = key->l3_dst.ip4;
```

Comment:
As this is only used for `dp_vnf_lbprefix_exists()` I guess just add the test for `l3_type` in the condition?

Reply:
Adding the l3_type for comparison is fine.